pkg/malwaremanager/v1/malware_manager_test.go

```
@@ -0,0 +1,271 @@
+		k8sClient:       nil,
+		nodeName:        "test-node",
+		clusterName:     "test-cluster",
+		metrics:         metricsmanager.NewMetricsMock(),
+		k8sObjectCache:  &objectcache.K8sObjectCacheMock{},
+	}
+}
+
+func TestCreateMalwareManager_NoClamAV(t *testing.T) {
+	orig, present := os.LookupEnv("CLAMAV_SOCKET")
+	if present {
+		os.Unsetenv("CLAMAV_SOCKET")
```

golangci-lint (2.12.2)

[error] Error return value of `os.Unsetenv` is not checked

(errcheck)